docs/snippets/quickstart-prereqs.md

```diff
@@ -0,0 +1,22 @@
+<!-- Snippet used in the following topics:
+- /docs/getting-started/README.md
+- /docs/install/quickstart-install.md
```

Comment:
I think this snippet is now on `/docs/getting-started/quickstart-install.md` as well now